### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by guirecordon**
**Responsive Design - Collaborative project:** 

In this project, we build a clone of The Next Web webpage, with a special focus on its responsiveness. We used mainly the CSS grid in our media queries to layout the design. We used a mobile-first approach with breakpoints. Following are some specifications of our project: 

- breaking points at 768px and 1024px;
- background images for most of the articles thumbnails; 
- used gradient for the hero image;
- used 'display: none' to adjust changes to the different screen sizes;
- used CSS flex to align the nav bar, etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,55 @@
-# tnw-page-mockup
-Building with a responsive design - a Microverse/The Odin Project collaborative project.
+# Building with Responsive Design [Collaborative Project]
+
+> This project takes a mobile first approach and consists of replicating the The Next Web website putting emphasis on how it behaves differently depending on the size of the screen where the website is rendered. [the next web website](https://thenextweb.com/).
+>
```
Comment (majovanilla) [inline README.md]: Great description!! :+1:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,738 @@
...
+      <div class="ta-atcl-credits">
+        <p>by <a href="#">Thomas Macaulay</a></p>
+        <p class="icon-adjust-flex">
+          <i class="material-icons">access_time</i>
+          2 hours ago
+          <span class="share-icon"><i class="material-icons ">share</i> 3</span>
+        </p>
+      </div>
+    </article>
+  </div>
```
Comment (majovanilla) [inline index.html]: The three articles in the top section are not showing the images. Check your link.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,738 @@
...
+  <div class="love-wrapper">
+    <section class="latest-news">
+      <h2 class="ln-heading">Latest news <i class="material-icons">keyboard_arrow_right</i></h2>
```
Comment (majovanilla) [inline index.html]: The articles in the latest news section are not showing the images. Check your links.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,898 @@
...
+
+.sg-img8 {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0.9)), url('../images/massive.jpg');
+}
+
+.sub-article img {
+  width: 96px;
+  height: 96px;
+}
+
```
Comment (majovanilla) [inline css/style.css]: These images look disproportioned in all screen sizes. Please fix it.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,738 @@
...
+        </div>
+      </section>
+    </div>
+    <section class="latest-deals">
+      <h2>Latest deals ></h2>
+      <div class="ln-mobile-atcl-wrapper">
+        <a href="#" class="ln-bg-img lt-dktp-img img9"></a>
+        <div class="ln-atcl-head">
+          <h3>The 2020 All-in-One Data Scientist Mega Bundle</h3>
+          <p><span class="line-through">$6000</span><span> - </span>$39.99</p>
```
Comment (majovanilla) [inline index.html]: The latest deals section is not displaying the images correctly. Please fix the URL in your CSS.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,738 @@
...
+      </div>
+
+    </section>
+    <section class="latest-funding">
+      <h2>Latest Funding rounds on index.co ></h2>
+      <ul class="lf-links">
+        <li>See More<span class="lf-links-bars"> | </span></li>
+        <li>Get Updates by email<span class="lf-links-bars"> | </span></li>
+        <li>Add Info</li>
+      </ul>
```
Comment (majovanilla) [inline index.html]: This section is not behaving exactly as the original one in the medium breakpoint. Check the images below, it's just a minor change in the way the subtitles are arranged. Yours look like this:
Screenshot from 2020-03-13 21-15-17

The original is:
Screenshot from 2020-03-13 21-15-23

**Review by majovanilla — CHANGES_REQUESTED** [review]
Hey team, 👋

## You did an *amazing*🌟 ✨ job on this project! 👊🏻  💯

- Good Readme file
- Good Pull Request Message


### Required Changes: ✍️
There are still some changes 👇 you need to implement to approve this request (see comments below) 🔍

### Optional Changes:

If you want to make your project even better 🔥💥, I have a few suggestions you can implement.  

These are completely [Optional]: 🦔

- Use semantic HTML in the structure of your HTML file. Favor the use of `<nav>`,  `<main>`,  `<footer>`, to enclose the elements on your file. You are already using nav and footer, just add the main tag to enclose the rest of your elements.

## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes

**Review by pelzolga123 — APPROVED** [review]
Approved.

**Comment by pelzolga123** [comment]
## Great  work, Congratulation :+1: Project Approved.
You did all changes required previously. Go ahead and merge to master.
_Goodluck in your next project_ *Happy coding*

**Pull request merged**